www/js/share/services/websocket.service.js

```diff
@@ -535,6 +535,18 @@ angular
                                 localStorage.landingCompany = message.authorize.landing_company_name;
                                 appStateService.scopes = message.authorize.scopes;
                                 amplitude.setUserId(message.authorize.loginid);
+                                // update accounts from account list whenever authorize is received
+                                const accounts = localStorage.getItem('accounts') && JSON.parse(localStorage.getItem('accounts'));
+                                const accountList = message.authorize.account_list;
+                                if (accounts && accounts.length) {
+                                    accounts.forEach((account, idx) => {
+                                        if (accountList) {
```

Comment:
it's better to move the `accounList` condition to line 451 in order to prevent a nonsense iterate over a `accounts`